contrib/Joshua/scripts/correctnessTest.sh

```diff
@@ -4,6 +4,6 @@
 export ASAN_OPTIONS="detect_leaks=0"
 
 OLDBINDIR="${OLDBINDIR:-/app/deploy/global_data/oldBinaries}"
-#mono bin/TestHarness.exe joshua-run "${OLDBINDIR}" false
+mono bin/TestHarness.exe joshua-run "${OLDBINDIR}" false
 
-python3 -m test_harness.app -s ${JOSHUA_SEED} --old-binaries-path ${OLDBINDIR}
+# python3 -m test_harness.app -s ${JOSHUA_SEED} --old-binaries-path ${OLDBINDIR}
```

Comment:
did you intend to push this as well?

Reply:
Yes. It's intended so our nightly can be useful.

Reply:
looking at the result from #8575 it seems that these fixes make the new test harness behave again. Not saying you shouldn't use the old test harness in a release branch for now, just wanted to point this out.